frontend/src/studentPage/HomeworkPage.tsx

```diff
@@ -31,10 +31,10 @@ export default function HomeworkPage(props : Readonly<Props>){
         <>
             <StudentNavbar  studentId={props?.studentId ?? ''} logout={props.logout} setLessons={setLessons}/>
             <div className="container">
+
+                <h3>Homework List</h3>
                 <br/>
                 <br/>
```

Comment:
html-elements shouldn't be used for styling. You could add css-styling to the header instead and add some bottom-margin to it.